lib/throttle.js

```diff
@@ -25,6 +25,21 @@ module.exports = (api) => {
     };
   };
 
+  api.metasync.debounce = (
+    timeout,
+    fn,
+    args
+  ) => {
+    let timer = null;
+    return () => {
+      clearTimeout(timer);
+      timer = setTimeout(
+        () => (args !== undefined ? fn(...args) : fn()),
+        timeout
```

Comment:
OTOH, I may be wrong, as far as I can see. For example, the documentation for `debounce` package from npm says:

> Creates and returns a new debounced version of the passed function that will postpone its execution until after wait milliseconds have elapsed **since the last time it was invoked**.